configure.ac

```diff
@@ -637,6 +637,34 @@
         exit 1
     fi
 
+    PCRE2=""
+    AC_CHECK_LIB(pcre2-8, pcre2_compile_8,,PCRE2="no")
+    if test "$PCRE2" = "no"; then
+        echo
+        echo "   ERROR!  pcre2 library not found, go get it"
+        echo "   from www.pcre.org. Or from packages:"
+        echo "   Debian/Ubuntu: apt install libpcre2-dev"
+        echo "   Fedora: dnf install pcre2-devel"
+        echo "   CentOS/RHEL: yum install pcre2-devel"
+        echo
+        exit 1
+    fi
+
+    AC_DEFINE([PCRE2_CODE_UNIT_WIDTH], [8], [Pcre code unit width is 8 bits])
+    AC_MSG_CHECKING(for PCRE2 JIT support)
+    AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[ #include <pcre2.h> ]],
+        [[
+        int jit = 0;
+        pcre2_config(PCRE2_CONFIG_JIT, &jit);
+        ]])],[ pcre2_jit_available=yes ],[ pcre2_jit_available=no ]
+        )
+    if test "x$pcre2_jit_available" = "xyes"; then
+        AC_MSG_RESULT(yes)
+        AC_DEFINE([PCRE2_HAVE_JIT], [1], [Pcre2 with JIT compiler support enabled])
+    else
+        AC_MSG_RESULT(no)
+    fi
+
     # libpcre 8.35 (especially on debian) has a known issue that results in segfaults
     # see https://redmine.openinfosecfoundation.org/issues/1693
     if test "$with_libpcre_libraries" = "no"; then
```

Comment:
This is done in 1986aa95278702f78923efc3b7dfb87d64fa5a27

This last commit is not part of the PR because CI fails if I include it in this PR, because commit-check cannot compile the first commits without PCRE1

Reply:
Not sure I get this. If you remove it last it should work, right?

Reply:
If I remember correctly, commit check will fail because it cannot build the first commits without PCRE1, and it uses only the latest `.github` dependency
Let's check in next PR

Reply:
commit check indeed fails.
I can split the last commit removing PCRE1 in two :
- the last part only for CI `.github` files, in a next PR
- the first part about configure.ac and everything else

Reply:
Discussion continued in https://github.com/OISF/suricata/pull/6394#issuecomment-923678398